dev-client/src/screens/LocationScreens/components/soilInfo/PropertiesScoreDisplay.tsx

```diff
@@ -50,6 +52,9 @@ export function PropertiesScoreDisplay({
     () => rowsFromSoilIdData(match.soilInfo.soilData),
     [match.soilInfo.soilData],
   );
+
+  const isOffline = useIsOffline();
```

Comment:
Thought: Hmm this can be considered later, but I wonder if we want to use `useDebouncedIsOffline` or something so that if someone's connection is spotty, the UI isn't flickering? (But I think it's not as easy as "just use the debounced hook", as the UI would then allow users to make a change that they shouldn't be able to make offline)